website/docs/r/autoscale_setting.html.markdown

```diff
@@ -0,0 +1,826 @@
+  } 
+ 
+  network_profile { 
+    name    = "terraformnetworkprofile" 
+    primary = true 
+ 
+    ip_configuration { 
+      name                                   = "TestIPConfiguration" 
+      subnet_id                              = "${azurerm_subnet.test.id}" 
+      load_balancer_backend_address_pool_ids = ["${azurerm_lb_backend_address_pool.bpepool.id}"] 
+      load_balancer_inbound_nat_rules_ids    = ["${element(azurerm_lb_nat_pool.lbnatpool.*.id, count.index)}"] 
+    } 
+  } 
+ 
+  tags { 
+    environment = "staging" 
+  } 
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the autoscale setting. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which to create the autoscale setting. Changing this 
+forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `enabled` - (Required) Specifies whether automatic scaling is enabled for the target resource.
+
+* `target_resource_id` - (Required) Specifies the resource ID of the resource that the autoscale setting should be added to.
+
+* `profile` - (Required) Specifies a collection of automatic scaling profiles that specify different scaling parameters for 
+different time periods. A maximum of 20 profiles can be specified.
+
+* `notification` - (Optional) Specifies notification settings for autoscale events.
+
+* `tags` - (Optional) A mapping of tags to assign to the resource.
+
+`profile` supports the following:
+
+* `name` - (Required) Specifies the name of the profile.
+
+* `capacity` - (Required) Specifies the number of instances that can be used during this profile.
+
+* `rule` - (Required) Specifies a collection of rules that provide the triggers and parameters for the scaling action. 
+A maximum of 10 rules can be specified.
+
+* `fixed_date` - (Optional) Specifies a date for the profile. This element is not used if the `recurrence` element is used.
+
+* `recurrence` - (Optional) Specifies the repeating times at which this profile begins. This element is not used if the `fixed_date` element is used.
+
+`capacity` supports the following:
+
+* `minimum` - (Required) Specifies the minimum number of instances that are available for the scaling action.
+
+* `maximum` - (Required) Specifies the maximum number of instances that are available for the scaling action. 
+The maximum number of instances is limited by the cores that are available in the subscription. You can use the portal to see 
+the number of cores that are available in your subscription.
+
+* `default` - (Required) Specifies the number of instances that are available for scaling if metrics are not available for 
+evaluation. The default is only used if the current instance count is lower than the default.
+
```

Comment:
Can you add a HR between this section and all of the following sections as well?
```markdown
---
```